### PR TITLE
Adds support for bastion volume using snapshot id

### PR DESCRIPTION
**Pull request by pdmars**

### CODE REVIEW

File: aws/aws_instance.go [hunk 1]
```diff
@@ -226,12 +228,23 @@ func (awsI *Ec2Instance) runTerraformApply(volumeSize int, zookeeperSize int, ma
 		return err
 	}
 
+	if (bastionVolSnapshotId != "") {
```
Comment (buzztroll) [inline aws/aws_instance.go]: Ideally this check would be near the logic that places all the other assets in newAwsDeploymentDescription.  Is that possible?  If not can we move it up to CreateInstance()?

File: aws/etc/terraform/instance/bastion.tf [hunk 2]
```diff
@@ -1,42 +1,31 @@
+resource "aws_instance" "bastion" {
+  availability_zone = "${data.aws_availability_zones.available.names[0]}"
```
Comment (buzztroll) [inline aws/etc/terraform/instance/bastion.tf]: Did we determine that offest 0 is always the same?

Comment (pdmars) [reply aws/etc/terraform/instance/bastion.tf]: yes, afaict, although it still deletes the bastion and recreates it so it doesn't break anything if it were to change AZs

**Comment by pdmars** [comment]
@buzztroll ok, addressed your comments and verified deploys both with and without this new option

**Pull request merged**